packages/common/src/utils/conditionalFormatExpressions.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Utilities for evaluating conditional expressions in format strings with parameters
+ */
+
+/**
+ * Strips the ld.parameters. prefix from a parameter name if present
+ * Examples:
+ *   - "ld.parameters.currency" -> "currency"
+ *   - "currency" -> "currency"
+ */
+function stripParameterPrefix(paramName: string): string {
+    const ldPrefix = 'ld.parameters.';
+    const lightdashPrefix = 'lightdash.parameters.';
```

Comment:
Should these be separate constants in their own module constants? At least the `ld|lightdash.parameters` parts